topics/chatgpt-api/index.md

```diff
@@ -0,0 +1,12 @@
+---
+aliases: implementation
+created_by: OpenAI
+display_name: ChatGPT API
+logo: chatgpt-api.png
+related: artificial-intelligence, machine-learning, natural-language-processing, chatgpt, openai
+released: 2022
+short_description: ChatGPT API is a RESTful API that provides a simple interface to interact with OpenAI's GPT-3 and GPT-Neo language models.
+topic: chatgpt-api
+url: https://github.com/sooftware/chat-gpt-api
```

Comment:
This URL (a) should be a `github_url`, the `url` field should be to ChatGPT's docs or landing page; and (b) doesn't seem to exist.